inspector/ios-inspector/dist/build_steps/sb_parse.json

```diff
@@ -0,0 +1,14 @@
+[
+    { "do": { "add_ios_system_framework": "AudioToolbox.framework" } },
+    { "do": { "add_ios_system_framework": "Accounts.framework" } },
+    { "do": { "add_ios_system_framework": "CFNetwork.framework" } },
+    { "do": { "add_ios_system_framework": "CoreGraphics.framework" } },
+    { "do": { "add_ios_system_framework": "libsqlite3.dylib" } },
+    { "do": { "add_ios_system_framework": "libz.1.1.3.dylib" } },
+    { "do": { "add_ios_system_framework": "MobileCoreServices.framework" } },
+    { "do": { "add_ios_system_framework": "QuartzCore.framework" } },
+    { "do": { "add_ios_system_framework": "Security.framework" } },
+    { "do": { "add_ios_system_framework": "Social.framework" } },
+    { "do": { "add_ios_system_framework": "StoreKit.framework" } },
+    { "do": { "add_ios_system_framework": "SystemConfiguration.framework" } }
+]
```

Comment:
Why are those additions necessary?

Reply:
Fixed in [this commit](https://github.com/Staffbase/trigger.io-parse/pull/10/commits/8984c2625c92a793b02d28c78c4a65dff9c59cfa).